### PR TITLE
[zh] Update admission-controllers.md

### PR DESCRIPTION
**Pull request by ptux**
related PR：
https://github.com/kubernetes/website/pull/30741

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @tengqm

**Comment by mengjiao-liu** [comment]
Please synchronize the Chinese version of the document after the English version of the document is merged.
/hold

**Comment by ptux** [comment]
> Please synchronize the Chinese version of the document after the English version of the document is merged.

Done

**Comment by mengjiao-liu** [comment]
Thank you for proposing this fix. However, we will not merge partial fixes. To find out what are the upstream changes to be synced, do this on the main branch:
```
./scripts/lsync.sh content/zh/docs/reference/access-authn-authz/admission-controllers.md
```
```diff
diff --git a/content/en/docs/reference/access-authn-authz/admission-controllers.md b/content/en/docs/reference/access-authn-authz/admission-controllers.md
index 324de2e0d..6f8bf5b8b 100644
--- a/content/en/docs/reference/access-authn-authz/admission-controllers.md
+++ b/content/en/docs/reference/access-authn-authz/admission-controllers.md
@@ -30,9 +30,9 @@ mutating and validating (respectively)
 which are configured in the API.
 
 Admission controllers may be "validating", "mutating", or both. Mutating
-controllers may modify the objects they admit; validating controllers may not.
+controllers may modify related objects to the requests they admit; validating controllers may not.
 
-Admission controllers limit requests to create, delete, modify or connect to (proxy). They do not support read requests.
+Admission controllers limit requests to create, delete, modify objects or connect to proxy. They do not limit requests to read objects.
 
 The admission control process proceeds in two phases. In the first phase,
 mutating admission controllers are run. In the second phase, validating
@@ -583,7 +583,8 @@ subresource of the referenced *owner* can change it.
 This admission controller implements additional validations for checking incoming `PersistentVolumeClaim` resize requests.
 
 {{< note >}}
-Support for volume resizing is available as an alpha feature. Admins must set the feature gate `ExpandPersistentVolumes`
+Support for volume resizing is available as a beta feature. As a cluster administrator,
+you must ensure that the feature gate `ExpandPersistentVolumes` is set
 to `true` to enable resizing.
 {{< /note >}}
 
@@ -696,8 +697,22 @@ PodNodeSelector allows forcing pods to run on specifically labeled nodes. Also s
 admission plugin, which allows preventing pods from running on specifically tainted nodes.
 {{< /note >}}
 
+### PodSecurity {#podsecurity}
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+This is the replacement for the deprecated [PodSecurityPolicy](#podsecuritypolicy) admission controller
+defined in the next section. This admission controller acts on creation and modification of the pod and
+determines if it should be admitted based on the requested security context and the 
+[Pod Security Standards](/docs/concepts/security/pod-security-standards/).
+
+See the [Pod Security Admission documentation](/docs/concepts/security/pod-security-admission/)
+for more information.
+
 ### PodSecurityPolicy {#podsecuritypolicy}
 
+{{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
+
 This admission controller acts on creation and modification of the pod and determines if it should be admitted
 based on the requested security context and the available Pod Security Policies.

```

**Comment by ptux** [comment]
@mengjiao-liu 

I don't know what to do.
Can't understand the guide.

I don't get the idea [英文原文的保留].

**Comment by mengjiao-liu** [comment]
> @mengjiao-liu
> 
> I don't know what to do. Can't understand the guide.
> 
> I don't get the idea [英文原文的保留].

 You need to update what you want to update at the same time, and the English and Chinese versions are out of sync, and then make them the content your PR needs to submit.

**Comment by mengjiao-liu** [comment]
Now that the English version of the content has been merged into the mian branch, you only need to sync the English version page to the Chinese page.

Otherwise, treating this PR as an incomplete PR will destroy our tracking of upstream changes.

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -51,7 +51,9 @@ which are configured in the API.
 
 <!--
 Admission controllers may be "validating", "mutating", or both. Mutating
-controllers may modify the objects they admit; validating controllers may not.
+controllers may modify related objects to the requests they admit; validating controllers may not.
+
+Admission controllers limit requests to create, delete, modify objects or connect to proxy. They do not limit requests to read objects.
```
Comment (ptux) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:56]: @mengjiao-liu 
I think I already synced with [30741](https://github.com/kubernetes/website/pull/30741).
What am I missing?

Comment (mengjiao-liu) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md:56]: Look at slack, I need to explain it to you.

**Comment by ptux** [comment]
## Suggestions

### Remove English contents from the zh docs.

[英文原文的保留](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E8%8B%B1%E6%96%87%E5%8E%9F%E6%96%87%E7%9A%84%E4%BF%9D%E7%95%99)
contraries to the following principles:

- Single Source of Truth
- One thing at a time

In this case, I have to sync the corresponding English contents which are not this PR's concern.
I can't 100% believe the page is synced so I have to check the  English contents anyway.
So what's the point of syncretizing English content?

> 为便于译文审查和变更追踪

cons might outweigh the pros.

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md (hunk 1 above)
Comment (ptux) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md:56]: thanks for the explanation.

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -1171,8 +1176,22 @@ PodNodeSelector 允许 Pod 强制在特定标签的节点上运行。
...
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+This is the replacement for the deprecated [PodSecurityPolicy](#podsecuritypolicy) admission controller
+defined in the next section. This admission controller acts on creation and modification of the pod and
+determines if it should be admitted based on the requested security context and the 
+[Pod Security Standards](/docs/concepts/security/pod-security-standards/).
+
+See the [Pod Security Admission documentation](/docs/concepts/security/pod-security-admission/)
+for more information.
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1189]: please translate this new section.

Comment (ptux) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md:1189]: I won't do that.

**Comment by ptux** [comment]
I close this PR cause can't follow the guideline for the zh contents.

**Pull request closed without merging**